README.md

```diff
@@ -70,13 +70,13 @@ Now you are ready to use them :)!
 We have a folder called **commands** where I added a command to create a new component with all folder structure. The way you can use it is running in the terminal inside commands folder `sh createComponent componentName` replacing componentName by your component. You have to write the component name in _camelCase_ notation.
```

Comment:
After following the instructions below, I felt like they are useful for someone already familiar with the project.
I would like to propose an alternative approach, that can be integrated or replace existing instructions, or we can leave it as maybe its just the way I work haha.

[Blueprint Intro Doc](https://docs.google.com/document/d/1Up3OSWbhMaFF3n1QlsxxSTznCX4k7x4HTtewxOF7PqA/edit?usp=sharing)

Reply:
Not a bad suggestion @bronz3beard. Always so good to improve on these instructions. Would you be able to open a PR with your suggested readme edits? I've been reading your G Doc and love it but not sure where you want to replace the content. Thank you so much! Such a help!

Reply:
Same as Lara mentioned, I think it is good, and it is always welcome some improvements in the docs, so if it is possible, could you create a PR with your updates please? thanks @bronz3beard